extensions/kpi.md

```diff
@@ -176,8 +175,7 @@ states:
         refName: callPulmonologist
         parameters:
           patientid: "{{ $.patientId }}"
-  end:
-    kind: terminate
+  end: true
```

Comment:
Here we are using `end: true` instead of `terminate: true` :thinking:

Reply:
good catch ill update